readme.md

```diff
@@ -33,6 +33,7 @@ Some good apps written with Electron.
 - [Sample apps](https://github.com/hokein/electron-sample-apps) - Various sample apps.
 - [Koko](https://github.com/hachibasu/koko) - IRC client.
 - [Leanote](https://github.com/leanote/desktop-app) - Cloud notepad.
+- [Loop Drop](https://github.com/mmckegg/loop-drop-app) - Live electronic music performance software.
```

Comment:
Should be added to the bottom.

Can you come up with a description without `software`?